pkg/reconciler/autoscaling/kpa/scaler_test.go

```diff
@@ -27,6 +27,7 @@ import (
 	"time"
 
 	// These are the fake informers we want setup.
+	filteredinformerfactory "knative.dev/pkg/client/injection/kube/informers/factory/filtered"
```

Comment:
the other imports here end in `fake` whereas this doesnt; I dont think this is actually a fake? should it be? ([client/injection/kube/informers/factory/filtered/fake](https://github.com/qibobo/pkg/blob/0fd6e9e556398bc4ed27440c48976066bdd79560/client/injection/kube/informers/factory/filtered/fake/fake_filtered_factory.go) does exist in the `pkg` PR)

Reply:
In the same vain: Should this make the other fake (and underscored) imports unnecessary?

Reply:
I think you are talking about the import changes in `kpa_test.go` instead of this file. 

For `kpa_test.go`,  I reduced one of the `underscored` fake import in the most recent commit. 

However,  the filteredinformerfactory is necessary, since the `WithSelectors` and `Labelkey` struct are not claimed in the fake folder.  Also,  it won't conflict with the underscore imports, since the `underscore` imports will do the register on kube fake client context only. 
``` 
	filteredinformerfactory "knative.dev/pkg/client/injection/kube/informers/factory/filtered"
	_ "knative.dev/pkg/client/injection/kube/informers/factory/filtered/fake"
```

Reply:
If it needs to import the non-fake package let's move the import out of the section of imports that starts with a comment saying "these are the fake informers" for clarity

Reply:
updated